modules/charon-core/src/main/java/org/wso2/charon3/core/attributes/MultiValuedAttribute.java

```diff
@@ -67,23 +69,23 @@ public Attribute getSubAttribute(String attributeName) throws CharonException {
      * @throws CharonException
      */
     @Override
-    public void deleteSubAttributes() throws CharonException {
+    public void deleteSubAttributes() {
         //here we delete the complex type sub attributes which act as sub values
         attributeValues.clear();
     }
 
-     /*
+    /*
      * clear all primitive values
      * @throws CharonException
      */
 
-    public void deletePrimitiveValues() throws CharonException {
+    public void deletePrimitiveValues() {
```

Comment:
Changing public method signatures is not recommended. There may be users who use this method already. Changes should be done in a backward-compatible manner.